src/include/daos_prop.h

```diff
@@ -458,6 +458,58 @@ daos_prop_merge(daos_prop_t *old_prop, daos_prop_t *new_prop);
 struct daos_prop_entry *
 daos_prop_entry_get(daos_prop_t *prop, uint32_t type);
 
+/**
+ * Set the string value of a property entry in a property. The property type must expect that it's
+ * entry is of a string type. This duplicates the string internally and the entry string is freed
+ * with daos_free_prop(). The user does not need to keep the string buffer around after this
+ * function is called. If the entry already has a string value set, it frees that and overwrites it
+ * with this new string.
+ *
+ * \param[in]		prop		Property list
+ * \param[in]		type		Type of property to look for
+ * \param[in]		str		String value to set in the prop entry
+ * \param[in]           len		Length of \a str
```

Comment:
Looks to be alignment issues here due to a mix of tabs and spaces.  Same comment for the other changes to this file.

Reply:
indeed. a copy paste issue. i will fix that and Kris's comment in a subsequent patch to this file where im adding another API, rather than wait for CI on just a comment change